src/swap/types.ts

```diff
@@ -207,6 +207,25 @@ export type SwapReact = {
   title?: string; // Title for the Swap component. (default: "Swap")
 };
 
+/**
+ * Note: exported as public Type
+ */
+export type SwapSettingsReact = {
+  children: React.ReactNode;
+  className?: string; // Optional className override for top div element.
+  icon?: ReactNode; // Optional icon override
+  text?: string; // Optional text override
+};
+
+/**
+ * Note: exported as public Type
+ */
+export type SwapSettingsSlippageReact = {
+  className?: string; // Optional className override for top div element.
+  // icon?: ReactNode; // Optional icon override
+  // text?: string; // Optional text override
```

Comment:
can these be removed?

Reply:
yep! In the middle of a refactor right now - pulling out SwapSettingsSlippage and not 100% sure what can stay and what can be removed